src/packages/client/src/__tests__/integration/happy/blog-env-postgresql/test.ts

```diff
@@ -53,10 +53,6 @@ test('Blog fixture: Postgres', async () => {
     ],
   })
 
-  // Make sure we're not leaking connection strings in node_modules
-
-  expect(prisma.internalDatasources).toBe(undefined)
```

Comment:
Why removing this? Is this done in another test now or not relevant anymore?

Reply:
We don't have them anymore.